Src/WitsmlExplorer.Api/Jobs/Common/LogCurvesReference.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace WitsmlExplorer.Api.Jobs.Common
+{
+    public class LogCurvesReference
+    {
+        public LogReference LogReference { get; set; }
+        public IEnumerable<string> Mnemonics { get; set; } = new List<string>();
```

Comment:
It seems to me we are intermixing words here mnemonics and curves, what do you think? Classname indicates Log and Curves. It might lead to less readability of the codebase.